docs/terminusdb.1.ronn

```diff
@@ -27,838 +27,14 @@ Display help regarding terminusdb.
 
 ### test
 
-`terminusdb test OPTIONS`
```

Comment:
What happened to all the remove docs in the manpages?

Reply:
oh that's interesting. This seems to be an issue with the automatic manpage generation. I'm not sure what's going on there.

Reply:
It's all coming from this automated commit: https://github.com/terminusdb/terminusdb/pull/1709/commits/a11d4aa7f54be8abf90412b439cdf4011d3ec182

Reply:
I see why it doesn't work. Your changes introduced a regression in markdown generation of the help page.

Try `./terminusdb help --markdown` in your branch vs `main`.

Reply:
Looks like the markdown generation wasn't able to handle a flag that didn't have a short version.
I hope I just fixed that.

Reply:
yup, manpage is back.